continuous-integration/starter-kits/compile-release-publish/java-maven/multimodule/multimodule-api/src/test/java/gov/nasa/jpl/ammos/ids/sample_projects/multimodule/api/NumericViewsTest.java

```diff
@@ -0,0 +1,59 @@
+package gov.nasa.ammos.ids.sample_projects.multimodule.api;
```

Comment:
Suggest refactoring the package name to be: `gov.nasa.jpl.ammos.ids.slim.sample_projects.multimodule.api`